images/cmd/start-custom.sh

```diff
@@ -262,7 +262,7 @@ if [ -d "$DIR" ]; then
   echo "Permissions for $DIR set to 700."
 fi
 
-
+ulimit -c 0 
```

Comment:
Should there be a comment in here for just quickly explaining what this does?

Reply:
sure can add that